commit a4ca26661f95a60974fb13a78b1a0c89f9c09ea9
Author: Willy Tarreau <w@1wt.eu>
Date:   Thu Jul 25 07:53:56 2019 +0200

    BUILD: threads: add the definition of PROTO_LOCK
    
    This one was added by commit daacf3664 ("BUG/MEDIUM: protocols: add a
    global lock for the init/deinit stuff") but I forgot to add it to the
    include file, breaking DEBUG_THREAD.
    
    (cherry picked from commit d6e0c03384cab2c72fb6ab841420045108ea4e6f)
    Signed-off-by: Christopher Faulet <cfaulet@haproxy.com>

diff --git a/include/common/hathreads.h b/include/common/hathreads.h
index a7c8dc93..b05215bd 100644
--- a/include/common/hathreads.h
+++ b/include/common/hathreads.h
@@ -562,6 +562,7 @@ enum lock_label {
 	AUTH_LOCK,
 	LOGSRV_LOCK,
 	DICT_LOCK,
+	PROTO_LOCK,
 	OTHER_LOCK,
 	LOCK_LABELS
 };
@@ -679,6 +680,7 @@ static inline const char *lock_label(enum lock_label label)
 	case AUTH_LOCK:            return "AUTH";
 	case LOGSRV_LOCK:          return "LOGSRV";
 	case DICT_LOCK:            return "DICT";
+	case PROTO_LOCK:           return "PROTO";
 	case OTHER_LOCK:           return "OTHER";
 	case LOCK_LABELS:          break; /* keep compiler happy */
 	};
